lua/lvim/core/commands.lua

```diff
@@ -29,6 +29,19 @@ M.defaults = {
       require("lvim.core.info").toggle_popup(vim.bo.filetype)
     end,
   },
+  {
+    name = "LvimDocs",
+    fn = function()
+      local documentation_url = "https://www.lunarvim.org/docs/installation"
+      if vim.fn.has "mac" == 1 or vim.fn.has "macunix" == 1 then
+        vim.fn.execute("!open " .. documentation_url)
+      elseif vim.fn.has "win32" == 1 or vim.fn.has "win64" == 1 then
+        vim.fn.execute("!start " .. documentation_url)
+      else
+        vim.fn.execute("!xdg-open " .. documentation_url)
```

Comment:
```suggestion
      elseif vim.fn.has "unix" == 1 then
        vim.fn.execute("!xdg-open " .. documentation_url)
      else
        vim.notify "Opening docs in browser is not supported on your OS"

```

Reply:
@LostNeophyte Great suggestion! Let's include it 👍

Reply:
Sorry, I added a new line on the end there. Could you remove it?

Reply:
No worries! Removed in dfbd885